### DISCUSSION TITLE
How can I complete typescript definitions for Carbon 11?

### DISCUSSION THREAD

**Discussion by br4e** [carbon-design-system/carbon#11239] [Help] [answered]
I get the following error when using Carbon 11 in a typescript file: 
<img width="438" alt="Screen Shot 2022-04-15 at 9 13 20 PM" src="https://user-images.githubusercontent.com/68077134/163660936-4f3e3e99-1192-46bb-b8e0-2a07bc9b22ba.png">

and there is no obvious typescript file for the package: 
<img width="590" alt="Screen Shot 2022-04-15 at 9 14 32 PM" src="https://user-images.githubusercontent.com/68077134/163660965-676a15aa-f043-40fb-a212-bc0154519470.png">

**Comment by tay1orjones** [comment, marked as answer]
See https://github.com/carbon-design-system/carbon/discussions/10752
